src/main/java/com/minecolonies/apiimp/CommonMinecoloniesAPIImpl.java

```diff
@@ -52,34 +54,36 @@
+    private final IGlobalResearchTree                    globalResearchTree     = new GlobalResearchTree();
+    private final IColonyManager                         colonyManager          = new ColonyManager();
+    private final ICitizenDataManager                    citizenDataManager     = new CitizenDataManager();
+    private final IMobAIRegistry                         mobAIRegistry          = new MobAIRegistry();
+    private final IPathNavigateRegistry                  pathNavigateRegistry   = new PathNavigateRegistry();
+    private final IBuildingDataManager                   buildingDataManager    = new BuildingDataManager();
+    private final IJobDataManager                        jobDataManager         = new JobDataManager();
+    private final IGuardTypeDataManager                  guardTypeDataManager   = new com.minecolonies.core.colony.buildings.registry.GuardTypeDataManager();
+    private final IInteractionResponseHandlerDataManager interactionDataManager = new InteractionResponseHandlerManager();
+    private final EventBus                               modEventBus            = new DefaultEventBus();
+
+    private IForgeRegistry<EquipmentTypeEntry>                         equipmentTypeRegistry;
+    private IForgeRegistry<BuildingEntry>                              buildingRegistry;
+    private IForgeRegistry<FieldRegistries.FieldEntry>                 fieldRegistry;
+    private IForgeRegistry<JobEntry>                                   jobRegistry;
+    private IForgeRegistry<GuardType>                                  guardTypeRegistry;
+    private IForgeRegistry<InteractionResponseHandlerEntry>            interactionHandlerRegistry;
+    private IForgeRegistry<ColonyEventTypeRegistryEntry>               colonyEventRegistry;
+    private IForgeRegistry<ColonyEventDescriptionTypeRegistryEntry>    colonyEventDescriptionRegistry;
+    private IForgeRegistry<ResearchRequirementEntry>                   researchRequirementRegistry;
+    private IForgeRegistry<ResearchEffectEntry>                        researchEffectRegistry;
+    private IForgeRegistry<ResearchCostType>                           researchCostRegistry;
+    private IForgeRegistry<RecipeTypeEntry>                            recipeTypeEntryRegistry;
+    private IForgeRegistry<CraftingType>                               craftingTypeRegistry;
+    private IForgeRegistry<QuestRegistries.ObjectiveEntry>             questObjectiveRegistry;
+    private IForgeRegistry<QuestRegistries.RewardEntry>                questRewardRegistry;
+    private IForgeRegistry<QuestRegistries.TriggerEntry>               questTriggerRegistry;
+    private IForgeRegistry<QuestRegistries.DialogueAnswerEntry>        questDialogueAnswerRegistry;
+    private IForgeRegistry<HappinessRegistry.HappinessFactorTypeEntry> happinessFactorTypeRegistry;
+    private IForgeRegistry<HappinessRegistry.HappinessFunctionEntry>   happinessFunctionRegistry;
```

Comment:
I mean can we generally avoid unnecessary whitespace changes? _or ideally finally make proper autoformat_ :D

Reply:
I mean we technically have a format guideline, it just wasn't applied to this file :D